app/src/main/java/com/pluu/webtoon/usecase/EpisodeyUseCase.kt

```diff
@@ -0,0 +1,21 @@
+package com.pluu.webtoon.usecase
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Pluu/WebToon/blob/ae64ac2267c99cb98c2fd2ab1b680508d11b226a/app/src/main/java/com/pluu/webtoon/usecase/EpisodeyUseCase.kt#L1"></span>class EpisodeListUseCase should be declared in a file named EpisodeListUseCase.kt (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>